ark/utils/segmentation_utils_test.py

```diff
@@ -192,3 +194,32 @@ def test_transform_expression_matrix_multiple_compartments():
     for cell in cell_data.cell_id:
         arcsinh_vals = np.arcsinh(cell_data.loc[:, cell, modified_cols].values)
         assert np.array_equal(arcsinh_data.loc[:, cell, modified_cols].values, arcsinh_vals)
+
+
+def test_visualize_segmentation():
+    with tempfile.TemporaryDirectory() as temp_dir:
```

Comment:
We should add a test to make sure multiple lists of channels produces the correct output (an overlay for each list of channels), and that if no channels are provided, only the expected images are produced